packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -127,6 +130,8 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
           }}
           data-automationid={'ColumnsHeaderColumn'}
           data-item-key={column.key}
+          onBlur={this._onColumnBlur}
```

Comment:
Does the tooltip need to appear on hover as well?

Reply:
it already does; the TooltipHost is wrapped around the column header's contents with no padding in between, so the mouse target works. The issue is that it doesn't include the focus target, which is the root node of the DetailsColumn component.